From 5118e532600549ad0f56cb9b1a179b8eab70c483 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Tue, 5 Apr 2016 05:19:05 -0700
Subject: [PATCH 18/25] Add a comment in memset-sse2-unaligned-erms.S

	* sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S: Add
	a comment on VMOVU and VMOVA.

(cherry picked from commit 696ac774847b80cf994438739478b0c3003b5958)
---
 sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S b/sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S
index 2deba42..4bf3d36 100644
--- a/sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S
+++ b/sysdeps/x86_64/multiarch/memset-sse2-unaligned-erms.S
@@ -1,6 +1,8 @@
 #if IS_IN (libc)
 # define VEC_SIZE	16
 # define VEC(i)		xmm##i
+/* Don't use movups and movaps since it will get larger nop paddings
+   for alignment.  */
 # define VMOVU		movdqu
 # define VMOVA		movdqa
 
-- 
2.8.1

